packages/graphql/src/schema/resolvers/wrapper.ts

```diff
@@ -32,6 +32,7 @@ import { getToken, parseBearerToken } from "../../utils/get-token";
 import type { SubscriptionConnectionContext, SubscriptionContext } from "./subscriptions/types";
 import { decodeToken, verifyGlobalAuthentication } from "./wrapper-utils";
 import type { Neo4jGraphQLSchemaModel } from "../../schema-model/Neo4jGraphQLSchemaModel";
+import { IncomingMessage } from "http";
```

Comment:
Would you be able to format this file with `prettier` and the prettier config provided in this repo? thanks!

Reply:
The changed files are now formatted with prettier in this PR, 864d3a94be987800114d78ac156a0bd09bd12f1d

Reply:
Perfect! Thanks!